proposals/tools.md

```diff
@@ -0,0 +1,90 @@
+# OCI Runtime-Tools and Image-Tools Project Proposals
+
+## Abstract
+The below proposal contemplates creation of two tools projects (runtime-tools and image-tools.)
+These would be associated with the OCI specification projects (runtime-spec and image-spec) and serve as repositories for testing  tools. 
+
+## Background
+Testing tools have been organically developed by the OCI developer community.
+These include:
+* Runtime-spec test code in a OCITools repository
+* Image-spec test code in the image-spec repository
+
+On Monday, August 22nd, 2016, the OCI Certification Program WG designated Chris Aniszczyk and Rob Dolin to draft a proposal for formalizing approval of tools.
+
+On Wednesday, August 24th, 2016, the OCI Developer Community (TDC) met and recommended establishing a tools repository corresponding to each of the spec repositories.  
+
+## Proposal
+With repositories under the http://github.com/OpenContainers organization:
+* Rename the OCITools repository as runtime-tools
+* Create a new repository named image-tools and populate this with the files from: https://github.com/opencontainers/image-spec/tree/master/cmd/oci-image-tool
+
+### Project Descriptions
+Below are brief draft project descriptions intended to serve as an initial abstract for each project:
+* Runtime-Tools: Tools for testing of container runtimes implementing the OCI runtime-spec.  
+This includes code that tests a runtime's conformance to the OCI Container Runtime spec.
+* Image-Tools: Tools for testing of container images implementing the OCI image-spec.  
+This includes code that validates a file's conformance to the OCI Container Image Format spec.  
+
+### Initial Maintainers
```

Comment:
On Wed, Aug 31, 2016 at 03:22:20PM -0700, Rob Dolin (MSFT) wrote:
> I would like to see OCI clarify how maintainers are added and
> removed after project creation since the OCI charter is a bit vague
> on this, but that's probably an issue for another pull request.

§5.b.viii.2 and .3 delegate maintainer addition/removal to the
(per-project?) TDC(s?).

The TDC-maintained (I think ;) project-template has those rules for
maintainer addition [1], although not all OCI Projects have adopted
them yet (by merging them into the per-project repo).  Removing
maintainers is not clear, but probably follows the two-thirds vote
dicussed in that doc for other purposes (it would be nice to spell it
out explicitly).

[1]: https://github.com/opencontainers/project-template/blob/52dbb39db883d7eeacf62865b028c4154578e5c3/MAINTAINERS_GUIDE.md#how-are-maintainers-added